packages/core/core/src/utils/startup-logger.ts

```diff
@@ -49,7 +48,7 @@ export default (app: Strapi) => {
 
       const addressTable = new CLITable();
 
-      const adminUrl = getAbsoluteAdminUrl(strapi.config);
+      const adminUrl = strapi.config.get('admin.absoluteUrl');
```

Comment:
```suggestion
      const adminUrl = strapi.config.get<string>('admin.absoluteUrl');
```

Reply:
And basically considering using this everywhere we get a value from the config so that we propagate correct types across the different part of the code

Reply:
The DX squad hasn't connected the typings to strapi.confg and should instead as it will actaully solve this in a better way

Reply:
oh, I thought it had been done in https://github.com/strapi/strapi/pull/17612
Make sense then